src/shogun/neuralnets/NeuralConvolutionalLayer.cpp

```diff
@@ -280,6 +292,10 @@ void CNeuralConvolutionalLayer::init()
 	SG_ADD(&m_pooling_height, "pooling_height", "Pooling Height", MS_NOT_AVAILABLE);
 	SG_ADD(&m_stride_x, "stride_x", "X Stride", MS_NOT_AVAILABLE);
 	SG_ADD(&m_stride_y, "stride_y", "Y Stride", MS_NOT_AVAILABLE);
+	
+	// SG_ADD(&m_initialization_mode, "initialization_mode", "Initialization Mode",
```

Comment:
Shall I typecast it to `machine_int_t*` like done below or make a new `Parameter::add` and a new entry in `lib/DataType.h`?